src/components/MessageList/MessageList.js

```diff
@@ -130,6 +132,11 @@ const MessageList = ({ messages, handleGoBack }) => {
     }
   };
 
+  const getUserAvatarURL = (username) => {
+    const host = process.env.REACT_APP_RC_HOST;
```

Comment:
There shouldn't be a concept of `env` variables when we deploy embeddedchat as a "component".

We get the host url from the props, I think we can use that.

We can store it in a separate store and use it directly here.

Reply:
Hm yeah I got it the[host ](https://github.com/RocketChat/EmbeddedChat/blob/a2157f97de26e66ac9749deee35e22b26f868257/src/index.js#L19) in index.js file in src folder right

Reply:
we can store Host before passing to `new RocketChatInstance`

Reply:
I think a better way to do it will be getting the host from the RocketChatInsance, create a separate fn called getHost and return the host there

Reply:
i made that function in lib/API

Reply:
Hey @sidmohanty11  can you please review the changes I made and let me know if any changes needed